app/uk/gov/hmrc/bindingtariffclassification/model/reporting/CaseReport.scala

```diff
@@ -24,12 +24,14 @@ case class CaseReport
 (
   filter: CaseReportFilter,
   group: CaseReportGroup,
-  field: CaseReportField
+  field: CaseReportField,
+  splitByType: Boolean = false
 )
 
 object CaseReport {
   private val reportGroupKey = "report_group"
   private val reportFieldKey = "report_field"
+  private val splitByTypeKey = "split_by_type"
```

Comment:
given `splitByType` is essentially a second group by, should we not change `group` to be a `Set[CaseReportGroup]`?

Reply:
I did this on a branch last week, I'll push it up to branch `groupbymulti`